Formula/zyre.rb

```diff
@@ -0,0 +1,61 @@
+# from https://github.com/Homebrew/homebrew-core/pull/6022/files
+class Zyre < Formula
+  desc "Local Area Clustering for Peer-to-Peer Applications"
+  homepage "https://github.com/zeromq/zyre"
+  url "https://github.com/zeromq/zyre/releases/download/v2.0.0/zyre-2.0.0.tar.gz"
+  sha256 "8735bdf11ad9bcdccd4c4fd05cebfbbaea8511e21376bc7ad22f3cbbc038e263"
+
+  head do
+    url "https://github.com/zeromq/zyre.git"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
+
+  option "with-drafts", "Build and install draft classes and methods"
```

Comment:
within the ZeroMQ "zproject" framework (a framework that auto-generates C based APIs based on XML doc, which generates the APIs and code around CZMQ and Zyre) there's the concept of a "draft API" which enables developers to include "alpha/beta" APIs within the various [stable] ZeroMQ frameworks. these API's are considered unstable, susceptible to change/breakage and are usually turned off by default in stable / production releases. so while messing with development it's nice to enable these API's (esp when working between the different underlying libraries such as CZMQ), in production mode they're generally defaulted to off so production code doesn't accidentally start using an API that's being tested.

there's a lot of C based code generation in the various ZMQ frameworks, this was a way to help automate some of the heavy lifting when it came to testing new APIs fast(er).

there's a lot more detail around how we do this here:

https://github.com/zeromq/zproject#draft-api-support

does that sorta make sense?